packages/arb-token-bridge-ui/src/styles/tailwind.css

```diff
@@ -18,6 +18,10 @@
   --blue-link: #1366C1;
 }
 
+*:focus-visible, button:focus-visible, input:focus-visible {
+  outline-color: white !important;
```

Comment:
tailwind css for `button:focus-visible, input:focus-visible` comes after this set of rules so i have to add `!important` to make sure it's obeyed